taxi/models.py

```diff
@@ -22,7 +22,9 @@ class Meta:
         verbose_name_plural = "drivers"
 
     def __str__(self):
-        return f"{self.username} ({self.first_name} {self.last_name})"
+        if self.first_name and self.last_name:
+            return f"{self.username} ({self.first_name} {self.last_name})"
+        return f"{self.username}"
```

Comment:
btw, if there is no `self.first_name` and `self.last_name`, then the __str__ method will return `self.username `, yes?. So do you really need these changes?

Reply:
Yes, because otherwise it will return `username ( )` or `username (first_name )` (with space)

Reply:
Yes, because otherwise the function will return `username ( )` or `username (first_name )` (with space). This is bad